triedb/pathdb/states.go

```diff
@@ -290,19 +285,14 @@ func (s *stateSet) revertTo(accountOrigin map[common.Hash][]byte, storageOrigin
 			if !ok {
 				panic(fmt.Sprintf("non-existent storage slot for reverting, %x-%x", addrHash, storageHash))
 			}
-			delta += len(blob) - len(data)
-
-			if len(blob) != 0 {
-				slots[storageHash] = blob
-			} else {
-				if len(data) == 0 {
-					panic(fmt.Sprintf("invalid storage slot mutation (null to null), %x-%x", addrHash, storageHash))
-				}
-				slots[storageHash] = nil
+			if len(blob) == 0 && len(data) == 0 {
+				panic(fmt.Sprintf("invalid storage slot mutation (null to null), %x-%x", addrHash, storageHash))
 			}
+			delta += len(blob) - len(data)
+			slots[storageHash] = blob
```

Comment:
So previously an empty blob would result in slots[hash] = nil, while it will be []byte now. 
I'm pretty sure its okay, but I just wanted to double check

Reply:
Yes, we don't really distinguish the nil and []byte{} here.